src/app/login/login.component.ts

```diff
@@ -26,7 +26,8 @@ export class LoginComponent implements OnInit {
     private spinner: NgxSpinnerService,
     private _route: ActivatedRoute,
     private _router: Router, private _userService: UserService) {
-    this.user = new User('', '', '', '', false);
+      this.user = new User('', '', '', '', '', false);
```

Comment:
Shame on you!
Haz que acepte vacío y pon esos valores vacíos por defecto en tu constructor. Eso caso de que quieras inicializar a cadenas vacías.
```
```